docs/buildah.md

```diff
@@ -0,0 +1,29 @@
+## buildah "March 2017"
+
+## NAME
+buildah - A command line tool to facilitate working with containers and their images. 
+
+
+## DESCRIPTION
+The buildah package provides a command line tool which can be used to:
+
+    * Create a working container, either from scratch or using an image as a starting point.
+    * Mount the working container's root filesystem for manipulation.
+    * Unmount the working container's root filesystem.
+    * Use the updated contents of the container's root filesystem as a filesystem layer to create a new image.
+    * Delete a working container or it's image.
+
+
+## SEE ALSO
+buildah-add(1)
```

Comment:
We need a one line comment on each one of these.

`buildah add`
          buildah add command adds the specified content to the container id.
...